playbooks/ansible-cloud/vcsa-appliance/files/bootstrap.yaml

```diff
@@ -1,4 +1,52 @@
 ---
+- hosts: controller
+  tasks:
+    - name: Add a vcenter entry in /etc/hosts
+      lineinfile:
+        dest: /etc/hosts
+        regexp: " vcenter "
+        line: "{{ hostvars['vmware-vcsa-6.7.0']['nodepool']['private_ipv4']}} vcenter.test vcenter"
+
+    - name: Setup base virtualenv_options
+      set_fact:
+        _virtualenv_options: "--python python{{ ansible_test_python }}"
+
+    - name: Enable --system-site-packages for virtualenv_options
+      set_fact:
+        _virtualenv_options: "{{ _virtualenv_options }} --system-site-packages"
+      when: ansible_os_family == "RedHat"
+
+    - name: Create virtualenv for ansible-test
+      shell: "virtualenv {{ _virtualenv_options }} ~/venv"
+
+    - name: Install ansible into virtualenv
+      shell: "~/venv/bin/pip install {{ ansible_user_dir }}/{{ zuul.projects['github.com/ansible/ansible'].src_dir }} -rtest/lib/ansible_test/_data/requirements/integration.cloud.vcenter.txt jmespath"
```

Comment:
we don't need to do this, as ansible-test will manage its own dependencies with the --requirements flag.